airflow/contrib/utils/kubernetes_utils.py

```diff
@@ -179,37 +179,47 @@ def dict_to_env(source, task_instance, context=None):
     return retval
 
 
-def uniquify_job_name(task_instance, context, run_timestamp=None, job_name=None):
+def uniquify_job_name(task, context, job_name=None):
```

Comment:
This entity that is passed is a task, and not a task instance - they are subtly different things, so I renamed the variable.